community/kernel/src/main/java/org/neo4j/kernel/NeoStoreDataSource.java

```diff
@@ -587,32 +587,25 @@ private NeoStoreModule buildNeoStore( final StoreFactory storeFactory, final Lab
             labelTokens, final RelationshipTypeTokenHolder relationshipTypeTokens,
             final PropertyKeyTokenHolder propertyKeyTokenHolder )
     {
-        final NeoStores neoStores = storeFactory.openNeoStores( true );
-
         life.add( new LifecycleAdapter()
         {
             @Override
             public void start() throws IOException
             {
-                // TODO: we should not need it anymore in case if we track ids during recovery,
-                // needs to be cleaned up in latest version
-                if ( startupStatistics.numberOfRecoveredTransactions() > 0 )
```

Comment:
Since we already touch this part lets cleanup all calculation of recovered transactions in `NeoStoreDataSource`? 
I can do it in separate pr, since it's not really related to main neo store refactory methods renaming thingy.